docs/claims.md

```diff
@@ -0,0 +1,175 @@
+download the latest version for your operating system. You can always find the most up-to-date
+releases of the desktop app on their [releases page][mycrypto].
+
+Once you've downloaded MyCrypto and have it running locally (we recommend an air-gapped computer for
+maximum security), you can start by navigating to the Claims app on Polkadot-JS Apps. Select the
+account you would like to claim the DOTs into and click the blue "Continue" button to proceed. Your
+screen should look something like this:
+
+![claim-1](assets/new-claims/claim-1.png)
+
+Now you will need to provide the Ethereum address that is associated with the DOT indicator tokens
+that you will claim. Enter the Ethereum address into the box and click "Continue". Next your screen
+should look the image below.
+
+![claim-2](assets/new-claims/claim-2.png)
+
+The hex encoded string that follows the sentence: "Pay DOTs to the Polkadot account:" is the
+hex-encoded public key of your Polkadot account, minus the `0x` prefix. To verify that the public
+key is correct you can use the `subkey` tool to inspect your address.
+
+The next step is to go to the MyCrypto application and click on "Sign & Verify Message" tab.
+
+![claim-3](assets/new-claims/claim-3.png)
+
+This will prompt you to select a method for unlocking your wallet.
+
+![claim-3](assets/new-claims/claim-4.png)
+
+After unlocking your wallet, you will copy and paste the outputted sentence into the input box.
+
+![claim-5](assets/new-claims/claim-5.png)
+
+When you click "Sign Message" you will get a JSON output like the below:
+
+![claim-6](assets/new-claims/claim-6.png)
+
+Copy and paste the JSON output of the signed message from MyCrypto into the input box on the
+Polkadot JS UI and click "Confirm Claim."
+
+![claim-7](assets/new-claims/claim-7.png)
+
+Now a green box will appear telling you the amount to claim with a "Claim" button to make the claim.
+Click on the "Claim" button and click "Submit (no signature)" to complete the claim.
+
+![claim-9](assets/new-claims/claim-9.png)
+
+At this point you will see a success message if everything went right and your DOTs will now be in
+the account that you claimed to.
+
+![claim-10](assets/new-claims/claim-10.png)
+
+Congratulations you can now participate in aspects of the Polkadot network such as
+[governance](learn-governance) and [staking](learn-staking). During the soft launch period balance
+transfers will not be enabled.
+
+#### Verifying your Claim
+
+After you make an on-chain claim for DOTs, your balance should be updated on the Polkadot UI
+immediately.
+
+Having trouble? Get support in the DOT [Claims Support][] channel.
+
+### Third Party Claims Processes
+
+\_COINBASE_TO_DO
+
+**We do not recommend using a third-party app or process to perform your claim or acquire DOT**
+
+Claiming using a third-party process can lead to the loss of your allocation, therefore we cannot
```

Comment:
How about "Claiming using other third-party processes..." instead?  Since we are directly endorsing Coinbase, above.